packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -394,6 +394,7 @@ Future<void> _writeAndroidPluginRegistrant(FlutterProject project, List<Plugin>
   );
   String templateContent;
   final AndroidEmbeddingVersion appEmbeddingVersion = _getAndroidEmbeddingVersion(project);
+  final bool isModule = project.isModule;
```

Comment:
Trivial nit: maybe move this closer to where is used.

Reply:
Sure, isModule happens to just be a pre-populated map key check, so it could be done in the loop